packages/pages-shared/environment-polyfills/types.ts

```diff
@@ -29,4 +29,9 @@ export type PolyfilledRuntimeEnvironment = {
 	Response: typeof Response;
 };
 
-export { fetch, Headers, Request, Response };
+export {
+	workerFetch as fetch,
+	WorkerHeaders as Headers,
+	WorkerRequest as Request,
+	WorkerResponse as Response,
+};
```

Comment:
That looks unintentional and possibly meaningfully different.

Reply:
yeah, this was an intentional change done in https://github.com/cloudflare/workers-sdk/commit/daa6c10815e90cd288282e9179729f5dbfe126c4. The way this code was previously written declared these variable typedefs in the global namespace, and then exported the (maybe-nonexistent) values using their globally declared names. This caused prettier to blow up trying to parse the file (`SyntaxError: Export 'fetch' is not defined. (32:9)`). Now, the exports reference the runtime value rather than the globally declared name. FWIW, these exports don't seem to actually be used anywhere.

Reply:
@GregBrimble are you ok with this? if so, can it be approved?